app/models/user.rb

```diff
@@ -0,0 +1,20 @@
+class User < ApplicationRecord
+  # de chuan hoa su ton tai cho name va email
+  VALID_EMAIL_REGEX = VALID_EMAIL_REGEX.Setting.freeze
+
+  before_save :downcase_email
+
+  validates :email, presence: true,
+    length: {minium: min_email.Setting, maxium: max_email.Setting},
+    format: {with: VALID_EMAIL_REGEX},
+    uniqueness: {case_sensitive: false}
```

Comment:
case_sensitive có tác dụng gì em ơi

Reply:
case_sensitivi để kiểm tra theo cả ký tự in hoa và in thường